content/e20co/drone-marathon/index.md

```diff
@@ -0,0 +1,103 @@
+---
+author: "dellintosh"
+date: 2017-01-06T18:00:00-06:00
+image: "e20co/drone-marathon"
+repo: "e20co/drone-marathon"
+tags: [ publish, marathon ]
+logo: marathon.svg
+title: marathon
+---
+
+This plugin can be used to deploy applications to a [Marathon](https://mesosphere.github.io/marathon/) server. The below pipeline configuration demonstrates simple usage:
+
+```yaml
+pipeline:
+  build:
```

Comment:
let's remove the `build` block in the samples for consistency with other plugin docs